### PR TITLE
Bootstrap install

### PR DESCRIPTION
**Pull request by Lydias303**
styling, added bootstrap, background, added font, logo and displayed trails as li's for the time being.

### CODE REVIEW

**Comment by bhargavisatpathy** [comment]
Looks great!

File: Brocfile.js [hunk 1]
```diff
@@ -4,9 +4,11 @@ var EmberApp = require('ember-cli/lib/broccoli/ember-app');
```
Comment (cluhring) [inline Brocfile.js:4]: what the hell is a Brocfile?

File: app/templates/application.hbs [hunk 2]
```diff
@@ -1,3 +1,5 @@
-<h1 id="title">Wonder Ways</h1>
+<img src="http://s24.postimg.org/u2976j7ol/transparent_wonder_ways_logo.png" id="logo"/>
```
Comment (cluhring) [inline app/templates/application.hbs:2]: Did you make that picture/drawing Lydia?  Is cool :)

File: config/environment.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ module.exports = function(environment) {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
         // e.g. 'with-controller': true
-      }
+      },
```
Comment (cluhring) [inline config/environment.js:13]: nice comma

File: app/styles/app.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
+body {
+  font-family: 'Lato', sans-serif;
+  background-image: url('images/mountain.jpg');
```
Comment (cluhring) [inline app/styles/app.css:3]: cool

**Pull request merged**